stdlib/3/hmac.pyi

```diff
@@ -0,0 +1,27 @@
+# Stubs for hmac (Python 3.5)
+#
+# NOTE: This dynamically typed stub was automatically generated by stubgen.
+
+from typing import Any
+from _operator import _compare_digest as compare_digest
```

Comment:
According to the unit tests there's no stub for `_operator` yet (nor for `operator`).